source/MaterialXView/Material.cpp

```diff
@@ -369,8 +399,14 @@ void Material::bindImages(mx::GLTextureHandlerPtr imageHandler, const mx::FileSe
         return;
     }
 
+    const std::string IMAGE_SEPERATOR("_");
+    const std::string UADDRESS_MODE_POST_FIX("_uaddressmode");
+    const std::string VADDRESS_MODE_POST_FIX("_vaddressmode");
+    const std::string FILTER_TYPE_POST_FIX("_filtertype");
+    const std::string DEFAULT_COLOR_POST_FIX("_default");
+    const int INVALID_MAPPED_INT_VALUE = -1; // Any value < 0 is not considered to be invalid
+
     const mx::VariableBlock* publicUniforms = getPublicUniforms();
-    mx::Color4 fallbackColor(0, 0, 0, 1);
```

Comment:
As above, it looks like we've accidentally lost an earlier fix to master in this merge.